prepare_zephyr.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash -xeu
+
+ZEPHYR_SHA="42fccba"
+ZEPHYR_SOURCE_URL="https://github.com/zephyrproject-rtos/zephyr/archive/${ZEPHYR_SHA}.zip"
+
+wget -Ozephyr.zip "$ZEPHYR_SOURCE_URL"
```

Comment:
add `--progress=dot:giga` to declutter the output

Reply:
Fixed.